pkg/serviceaccount/claims.go

```diff
@@ -47,12 +49,12 @@ type privateClaims struct {
 }
 
 type kubernetes struct {
-	ClusterName string          `json:"clusterName,omitempty"`
-	Namespace   string          `json:"namespace,omitempty"`
-	Svcacct     ref             `json:"serviceaccount,omitempty"`
-	Pod         *ref            `json:"pod,omitempty"`
-	Secret      *ref            `json:"secret,omitempty"`
-	WarnAfter   jwt.NumericDate `json:"warnafter,omitempty"`
+	ClusterName logicalcluster.LogicalCluster `json:"clusterName,omitempty"`
```

Comment:
(No action required) The persistent length of this everywhere maybe suggests consistent use of a shorter package alias?

Reply:
or just add an empty line